.claude/agents/jpa-analyzer.md

```
@@ -0,0 +1,602 @@
+user-invocable: true
+context: fork
+agent: Explore
+---
+
+# JPA Analyzer Agent
+
+JPA 코드에서 N+1 문제를 감지하고 쿼리 최적화 방안을 제시하는 분석 에이전트
+
+## 사용법
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)